packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -243,7 +243,7 @@ Generated by [AVA](https://avajs.dev).
         {
           encumberedBalance: {
             brand: Object @Alleged: USDC brand {},
-            value: 0n,
+            value: 750000n,
```

Comment:
Wonder why this changed 🤔

Reply:
same

Reply:
wait... does this mean that an advance didn't finish?

Reply:
The contract is restarted in the middle of the test using...

```js
  const kit = await EV.vat('bootstrap').consumeItem('fastUsdcKit');
  const actual = await EV(kit.adminFacet).restartContract(kit.privateArgs);
```

and the `privateArgs` in `fastUsdcKit` weren't updated with the new `feeConfig`